angular.json

```diff
@@ -56,8 +56,8 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "2mb",
-                  "maximumError": "5mb"
+                  "maximumWarning": "5mb",
+                  "maximumError": "8mb"
```

Comment:
This increase is unfortunate and there is intent to create an issue to reduce the size of css in the future